templates/docker-compose.liquid

```diff
@@ -64,6 +64,18 @@ services:
       - '{{dbport}}:{{dbport}}'
     networks:
       - {{name | downcase}}
+      
+  {{name | downcase}}Adminer:
+    container_name: {{name | downcase}}Adminer
+    restart: unless-stopped
+    ports:
+      - '1338:8080'
```

Comment:
could we keep this to 8080 possible as others might want to use port `1338` as a second strapi environment.
Other then that I'm happy to merge this.

Reply:
Sure, I chose this one  because 8080 might be crowded on some systems, but I have no opinion on that otherwise :)

Reply:
True we could do 9090:8080 good point

Reply:
all fixed up.